src/test_typing_extensions.py

```diff
@@ -1909,6 +1909,13 @@ def close(self):
 
         self.assertIsSubclass(B, Custom)
         self.assertNotIsSubclass(A, Custom)
+    def test_builtin_protocol_allowlist(self):
```

Comment:
Argh, sorry -- you _should_ have an empty line here (the issue was there used to be trailing whitespace). My mistake; I must have screwed up the GitHub suggestion

```suggestion

    def test_builtin_protocol_allowlist(self):
```

Reply:
Oh, I am not familiar with GitHub diff. I assumed, wrongly, that your previous suggestion delete the whitespaces only(instead of whole line).